css/components/navigation.scss

```diff
@@ -1 +1,29 @@
 // Navbar styles
+
+.navbar {
+  width: 85%;
+  height: auto;
+  line-height: 1.2;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-around;
+  background-color: $c__white;
+  @media (max-width: $b__breakpoint--large) {
```

Comment:
Always add spaces between the last style line and your media query, or nested styles (same for below)

```scss
.navbar {
  width: 85%;

  @media (max-width: 800px) {
    ...
  }

  @media (max-width: 640px) {
    ...
  }
}
```